src/erlcloud_inspector.erl

```diff
@@ -134,16 +134,16 @@ inspector_result_fun(#aws_request{response_type = error, error_type = aws} = Req
 -type attribute() :: proplists:proplist().
 
 -spec add_attributes_to_findings
-          (Attributes :: [attribute()],
-           FindingArns :: [string()]) ->
+          (Attributes :: attribute(),
```

Comment:
Is this type just mis-named? I feel like it should be `attributes()` in this case.

Reply:
This refers to a `proplist`. I didn't want to make considerations on naming, since it's out of the scope of the PR, but can change it if you prefer `attributes()` (mind you that, if exported, it'll break "interface").

Reply:
Nah, just leave it, as you say—the interface is what it is.